readme.md

```diff
@@ -0,0 +1,69 @@
+# Hackathon Project: SenaiLabIntegration
+
+## Overview
+
+Welcome to the GitHub repository for our project developed during the 20-hour InovaTI HACKATHON. This project aims to integrate a laboratory testing machine into the lab's digital system.
```

Comment:
I think that we could describe what the laboratory does whit one word in the beginning even whether it was described below, like: cement laboratory or construction site laboratory

Reply:
acknowledged